src/polymer/polymer-element.ts

```diff
@@ -221,14 +221,15 @@ function _getFlattenedAndResolvedBehaviors(
     //     array, which should be the most recently defined one.
     const foundBehavior = document.getOnlyAtId('behavior', behavior.name);
     if (!foundBehavior) {
-      throw new WarningCarryingException({
+      document.warnings.push({
```

Comment:
Only want to push this here if `document.url === behavior.sourceRange.file` because the behaviorAssignment could be in another file as part of a chain of behaviors. e.g.

`index.html`
```html
<link rel="import" href="foo.html">
<script>
  Polymer({is: 'index-el', behaviors: [Foo]});
</script>
```

`foo.html`
```html
<script>
  /** @polymerBehavior */
  const Foo = {behaviors: [NotFoundBehavior]};
</script>
```

So we don't want to report that NotFoundBehavior wasn't found in index.html, only in foo.html.